tox.ini

```diff
@@ -11,6 +11,8 @@ deps =
     nose
     unittest2
     protobuf==3.0.0-alpha-1
+setenv =
+    PYTHONPATH = {toxinidir}/_testing
```

Comment:
Ugh.  Don't we really want to be able to test *with* `gRPC` present, and wire in the workaround only if it is missing?